### PR TITLE
Add WSTR token metadata

### PR DESCRIPTION
**Pull request by silnem1**
ERC-20: https://etherscan.io/address/0xf0dc76c22139ab22618ddfb498be1283254612b1
NEP-141: https://explorer.mainnet.near.org/accounts/f0dc76c22139ab22618ddfb498be1283254612b1.factory.bridge.near
Aurora ERC-20: https://explorer.mainnet.aurora.dev/address/0xF34d508BaC379825255cc80F66CBc89DFeFF92E4

### CODE REVIEW

File: tokens/wstr.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
...0zOSA1MSAtNTggMTggLTMzIDE5IC03OSAxOSAtOTg1IGwwIC05NTAgNjczCjY3MSBjMzY5IDM3MCA2ODIgNjc3IDY5NCA2ODMgMjYgMTMgODMgMTQgMTE1IDIgMzYgLTE0IDc3IC02MCA5MCAtMTAyIDI2IC04OAo0OSAtNjEgLTY4NSAtNzk3IGwtNjcyIC02NzQgOTU1IDQgYzcwNSAyIDk2MCAwIDk3NSAtOSA5MSAtNTIgMTE1IC0xNjAgNTMKLTIzOCAtNDUgLTU3IC03IC01NSAtMTAzNSAtNTUgbC05NDggMCA2NzkgLTY3OSA2NzkgLTY3OSA1IC01NiBjNCAtNDggMSAtNjIKLTIxIC05NSAtNDUgLTY3IC0xMjQgLTg5IC0xOTUgLTUyIC0yMCAxMSAtMzMyIDMxNSAtNjk3IDY4MCBsLTY2MCA2NjEgLTUKLTk2NCAtNSAtOTY0IC0yNSAtMjcgYy03OCAtODMgLTIwMCAtNzEgLTI1MSAyNSAtMTggMzMgLTE5IDc5IC0xOSA5ODUgbDAgOTUwCi02NzIgLTY3MSBjLTM3MCAtMzcwIC02ODMgLTY3NyAtNjk1IC02ODMgLTMwIC0xNSAtODkgLTE0IC0xMjIgNCAtNDAgMjAgLTc5CjcwIC04NyAxMTAgLTE0IDc4IC0yMyA2OCA2OTAgNzgzIGw2NzEgNjczIC05NTAgLTIgYy02MTUgLTEgLTk1OSAyIC05NzUgOQotODIgMzMgLTExNyAxNDkgLTY3IDIyMyA0OSA3MyAtMTggNjkgMTA0MiA2OSBsOTUwIDAgLTY3MSA2NzMgYy0zNzAgMzY5IC02NzcKNjgyIC02ODMgNjk0IC0xNiAzMSAtMTQgOTAgNSAxMjkgMTggMzcgNjQgNzEgMTEyIDgwIDc0IDE1IDY5IDIwIDc4MCAtNjkwCmw2NzIgLTY3MSAwIDk1MSBjLTEgNTY4IDMgOTYwIDkgOTc1IDkgMjUgNDIgNTkgNzYgNzcgMzUgMjAgMTA5IDE0IDE0NSAtMTB6Ii8+CjxwYXRoIGQ9Ik0zNzkwIDk5OTYgYzAgLTIgOSAtNiAyMCAtOSAxMSAtMyAyMCAtMSAyMCA0IDAgNSAtOSA5IC0yMCA5IC0xMSAwCi0yMCAtMiAtMjAgLTR6Ii8+CjwvZz4KPC9zdmc+Cg==",
```
Comment (sept-en) [inline tokens/wstr.json:7]: It's better to use an optimized URL. Please use something like this [tool](https://yoksel.github.io/url-encoder/)

**Review by sept-en — CHANGES_REQUESTED** [review]

File: tokens/wstr.svg [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+-964 -5 -964 -25 -27 c-78 -83 -200 -71 -251 25 -18 33 -19 79 -19 985 l0 950
+-672 -671 c-370 -370 -683 -677 -695 -683 -30 -15 -89 -14 -122 4 -40 20 -79
+70 -87 110 -14 78 -23 68 690 783 l671 673 -950 -2 c-615 -1 -959 2 -975 9
+-82 33 -117 149 -67 223 49 73 -18 69 1042 69 l950 0 -671 673 c-370 369 -677
+682 -683 694 -16 31 -14 90 5 129 18 37 64 71 112 80 74 15 69 20 780 -690
+l672 -671 0 951 c-1 568 3 960 9 975 9 25 42 59 76 77 35 20 109 14 145 -10z"/>
+<path d="M3790 9996 c0 -2 9 -6 20 -9 11 -3 20 -1 20 4 0 5 -9 9 -20 9 -11 0
+-20 -2 -20 -4z"/>
+</g>
+</svg>
```
[the remark this answers is not in the document]
Comment (silnem1) [reply tokens/wstr.svg]: Updated with corrected image

File: tokens/wstr.json (hunk 1 above)
Comment (silnem1) [reply tokens/wstr.json]: Changed URL

File: tokens/wstr.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
...0.100000,-0.100000)'%0Afill='%23000000' stroke='none'%3E%3Cpath d='M0 6250 l0 -6250 6250 0 6250 0 0 6250 0 6250 -6250 0 -6250 0 0%0A-6250z m6351 2945 c15 -11 37 -35 50 -54 22 -32 24 -48 30 -210 4 -97 7 -637%0A8 -1201 l1 -1025 773 770 c424 424 811 808 858 853 l87 82 56 0 c63 0 98 -14%0A137 -57 74 -79 77 -140 12 -231 -26 -36 -403 -425 -845 -871 l-799 -806 1193%0A-5 c908 -4 1198 -8 1214 -17 66 -38 111 -131 99 -204 -8 -48 -57 -116 -93%0A-130 -52 -21 -406 -28 -1412 -29 l-1015 0 853 -853 852 -852 0 -63 c0 -96 -35%0A-154 -113 -187 -48 -20 -100 -19 -145 2 -23 11 -351 331 -868 845 -457 455%0A-833 828 -836 828 -2 0 -8 -541 -11 -1202 l-6 -1201 -28 -30 c-99 -103 -256%0A-88 -315 30 l-23 48 -3 1184 -4 1184 -771 -769 c-425 -424 -811 -807 -859%0A-853 l-88 -83 -63 4 c-75 4 -115 29 -158 96 -38 60 -38 104 0 164 47 73 258%0A291 980 1013 375 374 681 683 681 687 0 3 -537 8 -1193 10 -1111 3 -1195 4%0A-1219 20 -136 89 -112 307 37 339 34 8 423 13 1219 16 l1169 5 -769 771 c-423%0A425 -807 811 -852 858 -82 86 -82 87 -82 140 0 74 30 132 86 165 57 34 112 41%0A152 20 68 -35 187 -149 945 -905 l787 -785 0 944 c0 520 3 1064 7 1209 6 263%0A6 263 32 294 46 55 78 68 156 65 51 -2 77 -9 96 -23z'/%3E%3C/g%3E%3C/svg%3E",
```
Comment (sept-en) [inline tokens/wstr.json:7]:
```suggestion
    "icon": "data:image/svg+xml,%3Csvg version='1.0' xmlns='http://www.w3.org/2000/svg' width='1250.000000pt' height='1250.000000pt' viewBox='0 0 1250.000000 1250.000000' preserveAspectRatio='xMidYMid meet'%3E%3Cg transform='translate(0.000000,1250.000000) scale(0.100000,-0.100000)'%0Afill='%23000000' stroke='none'%3E%3Cpath d='M0 6250 l0 -6250 6250 0 6250 0 0 6250 0 6250 -6250 0 -6250 0 0%0A-6250z m6351 2945 c15 -11 37 -35 50 -54 22 -32 24 -48 30 -210 4 -97 7 -637%0A8 -1201 l1 -1025 773 770 c424 424 811 808 858 853 l87 82 56 0 c63 0 98 -14%0A137 -57 74 -79 77 -140 12 -231 -26 -36 -403 -425 -845 -871 l-799 -806 1193%0A-5 c908 -4 1198 -8 1214 -17 66 -38 111 -131 99 -204 -8 -48 -57 -116 -93%0A-130 -52 -21 -406 -28 -1412 -29 l-1015 0 853 -853 852 -852 0 -63 c0 -96 -35%0A-154 -113 -187 -48 -20 -100 -19 -145 2 -23 11 -351 331 -868 845 -457 455%0A-833 828 -836 828 -2 0 -8 -541 -11 -1202 l-6 -1201 -28 -30 c-99 -103 -256%0A-88 -315 30 l-23 48 -3 1184 -4 1184 -771 -769 c-425 -424 -811 -807 -859%0A-853 l-88 -83 -63 4 c-75 4 -115 29 -158 96 -38 60 -38 104 0 164 47 73 258%0A291 980 1013 375 374 681 683 681 687 0 3 -537 8 -1193 10 -1111 3 -1195 4%0A-1219 20 -136 89 -112 307 37 339 34 8 423 13 1219 16 l1169 5 -769 771 c-423%0A425 -807 811 -852 858 -82 86 -82 87 -82 140 0 74 30 132 86 165 57 34 112 41%0A152 20 68 -35 187 -149 945 -905 l787 -785 0 944 c0 520 3 1064 7 1209 6 263%0A6 263 32 294 46 55 78 68 156 65 51 -2 77 -9 96 -23z'/%3E%3C/g%3E%3C/svg%3E",
```

File: tokens/wstr.svg [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="1250.000000pt" height="1250.000000pt" viewBox="0 0 1250.000000 1250.000000"
+ preserveAspectRatio="xMidYMid meet">
+
```
Comment (sept-en) [inline tokens/wstr.svg:7]:
```suggestion
<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
 width="1250.000000pt" height="1250.000000pt" viewBox="0 0 1250.000000 1250.000000"
 preserveAspectRatio="xMidYMid meet">
```

**Review by sept-en — APPROVED** [review]

**Pull request merged**